src/language/utils.ts

```diff
@@ -491,3 +491,30 @@ export function isWhiteSpace(ch: number): boolean {
     // tslint:disable-next-line
     return (ts.isWhiteSpaceLike || (ts as any).isWhiteSpace)(ch);
 }
+
+export function convertModifiersToText(modifiers: ts.ModifiersArray | undefined, sourceFile?: ts.SourceFile): string {
+    if (modifiers === undefined || modifiers.length === 0) {
+        return "";
+    }
+
+    let text = "";
+
+    for (const modifier of modifiers) {
+        text += modifier.getText(sourceFile);
+    }
+
+    return `${text} `;
+}
+
+export function convertFunctionToArrowText(node: ts.FunctionExpression, sourceFile: ts.SourceFile): string {
+    const parenthesisIndex = sourceFile.text.indexOf("(", node.pos);
```

Comment:
Prefer `getChildOfKind` to avoid erroneous behavior if there is a comment that contains the search string